run--01a--prep-geo-data.Rmd

```diff
@@ -260,168 +280,279 @@ schoolShp <-
+    geom_sf(data = schoolShp %>% filter(school == "Bangor"), 
+            color = "green",   
+            fill = "green") +
+    geom_sf(data = schoolShp %>% filter(school == "Lewiston"), 
+            color = "yellow",   
+            fill = "yellow") +
+    geom_sf(data = schoolShp %>% filter(school == "Portland"), 
             color = "red",   
-            fill = NA) +
-    geom_sf(data = schoolShp %>% filter(school %in% c("Sanford")),
-            color = NA, 
-            fill = "green",
-            alpha = 0.2) + 
+            fill = "red") +
+    geom_sf(data = schoolShp %>% filter(school == "Sanford"), 
+            color = "purple",   
+            fill = "purple") +
+    geom_sf(data = schoolShp,
+            color = "black",   
+            fill = NA,
+            linewidth = 0.2) +
     theme_void()
+  
+  # Check with related county for each SDs
+  if (FALSE) {
+    ggplot() +
+      geom_sf(data = tractShp %>% filter(COUNTYFIP %in% c("001")),
+              color = "red",   
+              fill = NA) +
+      geom_sf(data = schoolShp %>% filter(school %in% c("Auburn")),
+              color = NA, 
+              fill = "green",
+              alpha = 0.2) + 
+      theme_void()
+    
+    ggplot() +
+      geom_sf(data = tractShp %>% filter(COUNTYFIP %in% c("019")),
+              color = "red",   
+              fill = NA) +
+      geom_sf(data = schoolShp %>% filter(school %in% c("Bangor")),
+              color = NA, 
+              fill = "green",
+              alpha = 0.2) + 
+      theme_void()
+    
+    ggplot() +
+      geom_sf(data = tractShp %>% filter(COUNTYFIP %in% c("001")),
+              color = "red",   
+              fill = NA) +
+      geom_sf(data = schoolShp %>% filter(school %in% c("Lewiston")),
+              color = NA, 
+              fill = "green",
+              alpha = 0.2) + 
+      theme_void()
+    
+    ggplot() +
+      geom_sf(data = tractShp %>% filter(COUNTYFIP %in% c("005")),
+              color = "red",   
+              fill = NA) +
+      geom_sf(data = schoolShp %>% filter(school %in% c("Portland")),
+              color = NA, 
+              fill = "green",
+              alpha = 0.2) + 
+      theme_void()
+    
+    ggplot() +
+      geom_sf(data = tractShp %>% filter(COUNTYFIP %in% c("031")),
+              color = "red",   
+              fill = NA) +
+      geom_sf(data = schoolShp %>% filter(school %in% c("Sanford")),
+              color = NA, 
+              fill = "green",
+              alpha = 0.2) + 
+      theme_void()
+  }
 }
 ```
 
 ```{r adds tract-SD crosswalk from NCES}
 # schoolShp downloaded from the Census does not contain tract information
 # Crosswalk can be downloaded from the National Center for Education Statistics
 # Reference: https://nces.ed.gov/programs/edge/geographic/relationshipfiles
-grffile <- glue("{input_path}GRF22/grf22_lea_tract.xlsx")
+# Note that school districts are updated every year
+grffile <- glue("{input_path}grf23_lea_tract.xlsx")
```

Comment:
I'm a bit nervous about this being hard-coded. I wouldn't hold up this PR for this, but I think it's worth creating an issue to generalize this.